prow/spyglass/viewers/viewers.go

```diff
@@ -146,7 +146,6 @@ func RegisterViewer(viewerName string, metadata ViewMetadata, handler ViewHandle
 	}
 	viewHandlerReg.reg[viewerName] = handler
 	viewMetadataReg.reg[viewerName] = metadata
-	logrus.Infof("Spyglass registered viewer %s with title %s.", viewerName, metadata.Title)
```

Comment:
maybe change the log level instead of removing it entirely? this seems useful to know